server/storage.ts

```
@@ -1612,6 +1618,236 @@ export class DatabaseStorage implements IStorage {
+      if (!repo) {
+        log(`No registered repository found with name: ${fullName}`, 'storage');
+        return null;
+      }
+
+      log(`Found registered repository: ${repo.id}`, 'storage');
+      return repo;
+    } catch (error) {
+      log(`Error finding repository by full name ${fullName}: ${error instanceof Error ? error.message : String(error)}`, 'storage');
+      throw error;
+    }
+  }
```

Biome (2.1.2)

[error] Duplicate class member name "findRepositoryByFullName"



(lint/suspicious/noDuplicateClassMembers)